src/backend/base/langflow/schema/message.py

```diff
@@ -65,13 +65,13 @@ def validate_flow_id(cls, value):
         return value
 
     @field_serializer("flow_id")
-    def serialize_flow_id(value):
+    def serialize_flow_id(self, value):
```

Comment:
Or could be a `staticmethod`. Using an instance method for consistency with pydantic docs.